python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -710,6 +758,13 @@ def _find_stars(self, exposure, background, id_generator):
 
         # Measure everything, and use those results to select only stars.
         self.star_measurement.run(sources, exposure)
+        # Run the normalization calibration flux task to apply the
+        # normalization correction to create normalized
+        # calibration fluxes.
+        self.star_normalized_calibration_flux.run(
+            exposure=exposure,
+            catalog=sources,
+        )
```

Comment:
I'd just put this all on one line, but it's up to you; we've got 110 chars here.